src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateMethod.cs

```diff
@@ -227,6 +227,14 @@ public override void ExecuteCmdlet()
             Mandatory = false)]
         public bool AutomaticOSUpgrade { get; set; }
 
+        [Parameter(
+            Mandatory = false)]
+        public SwitchParameter EnableAutoRollback { get; set; }
```

Comment:
@hyonholee same question about providing both of these parameters in the same parameter set